README.md

```diff
@@ -41,6 +41,8 @@ This will run the docker containers:
 
 ## Access the Development Container
 
```

Comment:
Add to the readme an explanation of how it is possible to map a .conan2 folder in order to have persistent data storage.